.readthedocs.yml

```diff
@@ -3,6 +3,13 @@ build:
   os: ubuntu-20.04
   tools:
     python: "3.12"
+  jobs:
+    post_checkout:
+      # unshallow so version can be derived from tag
+      - git fetch --unshallow || true
+    pre_build:
+      # run towncrier to preview the next version’s release notes
+      - "( find docs/release-notes -regex '[^.]+[.][^.]+.md' | grep -q . ) && towncrier build --keep"
```

Comment:
Sorry wait, what is the effect of this on our releases?

Reply:
It seems to change the file of the current version.  So if I run `towncrier build --version=0.10.9` with no files, it overrides the `0.10.9.md` file.  Similarly, this will start adding our `dev` notes to our `latest` docs, which I guess is ok?

Reply:
nothing. this line means “if there are fragments when RTD builds the docs, we create an ephemeral release notes file that doesn’t get committed anywhere, and then build the docs containing that pre-version’s release notes”

Reply:
I guess this really solidifies the fact that there should be no stray fragments after _any_ release on the release base branch.  I like it.